CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributer's Guide
+
+We welcome contributions from the community and have gathered guidelines 
+here to help you get started.
+
+## Discussion
+
+While not absolutely required, it is encouraged that you first open an issue 
+for any bug or feature request. This allows discussion on the proper course of
+action to take before coding begins.
+
+## Building
+
+To build from source you first need to `npm install`. You will also need a
+version of TypeScript available, either globally or install locally with
+`npm install typescript`. Then run `node build`.
```

Comment:
I'd simply this and make `typescript` a dev dependency. Then add an `npm` script for building the project. Here's my normal set up: https://github.com/blakeembrey/typescript-simple-loader/blob/master/package.json#L15-L21